offchain/hello-world-browser/src/Page/Home.purs

```diff
@@ -39,11 +42,16 @@ _doIncrement
   -> FundsLocked
   -> H.HalogenM State Action slots o m Unit
 _doIncrement datum fundsLocked = do
-  H.modify_ $ const (Incrementing datum (datum + 2))
+  let HelloWorldIncrement inc = helloWorldIncrement
+  H.modify_ $ const (Incrementing datum (datum + inc))
   result <- increment helloWorldIncrement
+  new <- getDatum
```

Comment:
should this be there only if `result` is a `Right`?

Reply:
Yup, thanks for catching that :)